src/components/board/board.test.js

```diff
@@ -1,8 +1,72 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer'; // ES6
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import Board from './board';
 
+// I need to test to make sure my board component contains 64 square components
+// arrange, act, assert
+// Arrange, your app is in a certain original state
+// Act, then something happens (click event, input, etc.)
+// Assert, or make a hypothesis, of the new state of your app
+// Your tests should test the functionality of the app, that mimic how it will be used by your end users.
```

Comment:
Yes! at some point this is how these tests will evolve to handle all those things on board. Nice break down.